docs/docs/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 - Add ability to set client certificates for downstream connections. [GH-259](https://github.com/pomerium/pomerium/pull/259)
 
+- Add ability to override HTTPS backend's TLS Server Name. [GH-297](https://github.com/pomerium/pomerium/pull/297)
```

Comment:
Do you mind merging the latest version of CHANGELOG and putting this entry above the [v3.0.0](https://github.com/pomerium/pomerium/blob/bc72d08ad443a1c7228d599c1cc099664bdf2f01/docs/docs/CHANGELOG.md#v030) fold?